plugins/c8y_configuration_plugin/src/upload.rs

```diff
@@ -177,9 +179,7 @@ pub async fn handle_config_upload_request_child_device(
 
 pub async fn handle_child_device_config_snapshot_response(
     message: &Message,
-    tmp_dir: &Path,
     http_client: &mut impl C8YHttpProxy,
-    local_http_host: &str,
     config_dir: &Path,
 ) -> Result<Message, anyhow::Error> {
     let config_response = ConfigOperationResponse::try_from(message)?;
```

Comment:
An ill-formed response from the child should be reported as a failure to the cloud.